docs: add notes on configuration for local dev (#186)
cipherstash/proxy: README.md

# Log format
# Optional
# Valid values: `pretty | text | structured (json)`

> # Default: `pretty` if Proxy detects it is running in a terminal context on startup (uses structured otherwise)

Reviewer: ```suggestion
# Default: `pretty` if Proxy detects during startup that a terminal is attached, otherwise `structured`
```

# Enable ansi (colored) output
# Optional

> # Default: `true` if Proxy detects it is running in a terminal context on startup

Reviewer: ```suggestion
# Default: `true` if Proxy detects during startup that a terminal is attached, otherwise `false`
```

port = "9930"
```

### Recommended settings for development

The default configuration settings are biased toward running in production environments.
Although Proxy attempts to detect the environment and set a sensible default for logging, your mileage may vary.

To turn on human-friendly logging:

```bash
CS_LOG__FORMAT = "pretty"
CS_LOG__ANSI_ENABLED = "true"
```

> If you are frequently changing the database schema or making updates to the column encryption configuration,
> it can be useful to reload the config and schema more frequently.

Reviewer: ```suggestion
If you are frequently changing the database schema or making updates to the column encryption configuration, it can be useful to reload the config and schema more frequently:
```
